Projects/App/Sources/Screens/ProfileScreen/ProfileState.swift

```diff
@@ -47,30 +48,33 @@ public final class ProfileStore: StateStore<ProfileState, ProfileAction> {
             return FiniteSignal { callback in
                 let disposeBag = DisposeBag()
 
-                let getProfileData = self.giraffe.client
+                let getChargeEstimationData = self.giraffe.client
                     .fetch(
-                        query: GiraffeGraphQL.ProfileQuery(),
+                        query: GiraffeGraphQL.ChargeEstimationQuery(),
                         cachePolicy: .fetchIgnoringCacheData
                     )
 
-                let getPartnerData = self.octopus.client
+                let getProfileData = self.octopus.client
                     .fetch(
-                        query: OctopusGraphQL.ParnerDataQuery(),
+                        query: OctopusGraphQL.ProfileQuery(),
                         cachePolicy: .fetchIgnoringCacheData
                     )
-                disposeBag += combineLatest(getProfileData.resultSignal, getPartnerData.resultSignal)
-                    .onValue { (profileData, partnerData) in
-                        if let profileData = profileData.value {
-                            let name = (profileData.member.firstName ?? "") + " " + (profileData.member.lastName ?? "")
-                            let monthlyNet = Int(
-                                profileData.chargeEstimation.subscription.fragments.monetaryAmountFragment
+                disposeBag += combineLatest(getChargeEstimationData.resultSignal, getProfileData.resultSignal)
+                    .onValue { (chargeEstimationData, profileData) in
+                        var monthlyNet = 0
+                        if let chargeEstimationData = chargeEstimationData.value {
+                            monthlyNet = Int(
+                                chargeEstimationData.chargeEstimation.subscription.fragments.monetaryAmountFragment
                                     .monetaryAmount.floatAmount
                             )
-                            callback(.value(.setProfileState(name: name, monthlyNet: monthlyNet)))
                         }
-                        if let partnerData = partnerData.value {
-                            let partner = PartnerData(with: partnerData.currentMember.fragments.partnerDataFragment)
+                        if let profileData = profileData.value {
+                            let name = profileData.currentMember.firstName + " " + profileData.currentMember.lastName
+                            let email = profileData.currentMember.email
+                            let partner = PartnerData(with: profileData.currentMember.fragments.partnerDataFragment)
                             callback(.value(.setEurobonusNumber(partnerData: partner)))
+                            callback(.value(.setMonthlyNet(monthlyNet: monthlyNet)))
```

Comment:
move to 
                        if let chargeEstimationData = chargeEstimationData.value {
like written in comment above